CMakeLists.txt

```diff
@@ -39,169 +40,192 @@ include(GNUInstallDirs)
 include(ProcessorCount)
 include(macros_and_options)
 
-option(USE_PYTHON "enable python support" FALSE)
-option(USE_PYTHON2 "if USE_PYTHON enabled, use python2 instead of python3" FALSE)
-option(BUILD_GLEW "build glew" FALSE)
+option(USE_PYTHON "enable python support" ON)
+option(USE_PYTHON2 "if USE_PYTHON enabled, use python2 instead of python3" OFF)
+
+
 
 ###############################################################################
+#########          Dependency Build Switches                   ################
+###############################################################################
+
+option(BUILD_ALL_DEPS "build ALL dependencies" ON)
+
+option(BUILD_GLEW "build glew"                OFF)
+option(BUILD_TBB "build TBB"                  OFF)
+option(BUILD_VDB "build OpenVDB"              OFF)
+option(BUILD_BOOST "build BOOST"              OFF)
+option(BUILD_OPENEXR "build OpenEXR"          OFF)
+option(BUILD_OPENIMAGEIO "build OpenImageIO"  OFF)
+option(BUILD_OPENCOLORIO "build OpenColorIO"  OFF)
+option(BUILD_PTEX        "build Ptex"          ON)
+option(BUILD_OPENSUBDIV  "build OpenSubdiv"    ON)
+
+if(BUILD_ALL_DEPS)
+#  set(BUILD_GLEW ON)
```

Comment:
?

Reply:
This way you are able to either build all dependencies or only a subset and use the system-wide installed versions for the others.

Reply:
This way you are able to either build all dependencies or only a subset and use the system-wide installed versions for the others.

Reply:
no, I meant why is build_glew the only dependency commented out?